apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -211,6 +218,87 @@ export class TeacherFeedback extends Component {
     return feedbackUnchanged;
   };
 
+  getFriendlyDate = feedbackSeen => {
```

Comment:
Since you aren't using `this`, this function should not be an arrow function and instead just be 
`function getFriendlyDate(feedbackSeen) { ...`
(details in [style guide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react) -> [airbnb styleguide](https://github.com/airbnb/javascript/tree/master/react#methods))